server/src/test/java/org/elasticsearch/search/SearchModuleTests.java

```diff
@@ -266,7 +272,8 @@ public void testRegisterPipelineAggregation() {
 
         assertThat(
                 module.getNamedXContents().stream()
-                    .filter(entry -> entry.categoryClass.equals(BaseAggregationBuilder.class) && entry.name.match("test"))
+                    .filter(entry -> entry.categoryClass.equals(BaseAggregationBuilder.class) &&
+                        entry.name.match("test", LoggingDeprecationHandler.INSTANCE))
```

Comment:
These might be better if they just fail rather than go to the DepercationLogger. This is fine though.

Reply:
I think for now this works and maybe once everything is cut over, we can make the single arity one use the unsupported operation exception version, either way, we can change this pretty easily later.